hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -86,7 +87,7 @@
  */
 public class SparkMain {
 
-  private static final Logger LOG = Logger.getLogger(SparkMain.class);
+  private static final Logger LOG = LogManager.getLogger(SparkMain.class);
```

Comment:
Just want to know why we have to change `Logger` to `LogManager` here (and also in other places) ?

Reply:
Hi @rahil-c ,
Logger belongs to log4j-1.2-api while LogManager is log4j 2.x API. Previously I changed all logs to slf4j but now restored all of them to log4j2.

Reply:
Thanks!